package.json

```diff
@@ -103,11 +110,13 @@
     "webpack-manifest-plugin": "^2.2.0"
   },
   "scripts": {
-    "ci-lint": "eslint --config static/.eslintrc --ignore-path static/.eslintignore ./static/*.js ./static/panes/*.js ./static/modes/*.js && eslint --config .eslintrc --ignore-path .eslintignore ./app.js ./lib/**",
+    "ci-lint": "npm run lint",
     "ci-test": "nyc --reporter=lcov mocha --recursive",
-    "lint-client": "eslint --config static/.eslintrc --ignore-path static/.eslintignore --fix ./static/*.js ./static/panes/*.js ./static/modes/*.js",
-    "lint-server": "eslint --config .eslintrc --ignore-path .eslintignore --fix ./app.js ./lib/**",
-    "lint": "npm run lint-client && npm run lint-server",
+    "lint-client": "eslint --config static/.eslintrc --ignore-path static/.eslintignore ./static/*.js ./static/panes/*.js ./static/modes/*.js",
+    "lint-server": "eslint --config .eslintrc --ignore-path .eslintignore ./app.js ./lib/**",
+    "lint-tests": "eslint --config test/.eslintrc --ignore-path test/.eslintignore ./test/**/*.js ./test/*.js",
+    "lint": "npm run lint-client && npm run lint-server && npm run lint-tests",
```

Comment:
the help string in `make lint` implies `--fix`, which is no longer the case.

We should either update the help string in the makefile or restore the `--fix` behavior of `make lint`